### PR TITLE
fix: :bug: #6969

### LINKED ISSUE #6969: Wrong network name at swap gas estimation hover
**Bug Description**
"Network fees are paid to the **Ethereum** network to secure transactions. Learn more".
While I was on Polygon

**Steps to Reproduce**

1. Visit swap page https://app.uniswap.org/#/tokens/polygon/0xb515790f12ad720d5fb256476fe7d443e3c5d752
2. Enter the amount to swap
3. Hover to gas icon (Network fees block)

**Expected Behavior**
"Network fees are paid to the **Polygon** network to secure transactions. Learn more".
etc

**Additional Context**
'ethereum'

### PR DESCRIPTION
**Pull request by lyzergic**
Inserted chain label instead of hardcoded "Ethereum" value

## Description
Fixes #6969 
Translation string had the name of the chain hardcoded, `Ethereum` which I changed. Changed the translations in Crowdin for Romanian, Italian and Russian since those are the ones I know. Let me know if I need to change all.

### Before
|    Mobile    |   Desktop    |
| ------------ | ------------ |
| paste_before | paste_after |

### After
|    Mobile    |   Desktop   |
| ------------ | ----------- |
| paste_after  | paste_after |

### Reproducing the error

1. 

### QA (ie manual testing)

- [ ] N/A

### Automated testing

- [ ] Unit test
- [ ] Integration/E2E test

### CODE REVIEW

File: src/components/swap/GasBreakdownTooltip.tsx [hunk 1]
```diff
@@ -83,7 +91,7 @@ export function GasBreakdownTooltip({
           <Trans>
             <InlineUniswapXGradient>UniswapX</InlineUniswapXGradient> aggregates liquidity sources for better prices and
             gas free swaps.
-          </Trans>{' '}
+          </Trans>
```
Comment (zzmp) [inline src/components/swap/GasBreakdownTooltip.tsx:94]: Why did you remove the space here?

File: src/components/swap/GasBreakdownTooltip.tsx [hunk 2]
```diff
@@ -9,6 +10,8 @@ import { isClassicTrade, isUniswapXTrade } from 'state/routing/utils'
 import styled from 'styled-components/macro'
 import { Divider, ExternalLink, ThemedText } from 'theme'
 
+import { getChainInfo } from '../../constants/chainInfo'
```
Comment (zzmp) [inline src/components/swap/GasBreakdownTooltip.tsx:13]: Import this as `import { getChainInfo } from 'constants/chainInfo'` to match the style of the file.
In general, import using a relative path when the file you're importing from is in the same subtree.

**Review by zzmp — CHANGES_REQUESTED** [review]
This looks great, thank you! Just a few quick questions and then I can run tests and merge this in :)

File: src/components/swap/GasBreakdownTooltip.tsx (hunk 2 above)
Comment (lyzergic) [reply src/components/swap/GasBreakdownTooltip.tsx:13]: done!

File: src/components/swap/GasBreakdownTooltip.tsx (hunk 1 above)
Comment (lyzergic) [reply src/components/swap/GasBreakdownTooltip.tsx:94]: I don't use these kind of spaces in my codebases and kinda did it by inertia. Checked and you are right, it served a purpose and looks better with it, so I reverted it.

**Comment by lyzergic** [comment]
@zzmp I changed the translations in most of languages( the ones  where it was clear where `Ethereum` stood in the sentence ), but now it asks me to be part of the team to be able to translate more.

**Review by zzmp — APPROVED** [review]

**Review by JackShort — APPROVED** [review]

**Comment by just-toby** [comment]
@lyzergic can you update to resolve the merge conflict?

**Comment by lyzergic** [comment]
It was fixed in another pr
